cylc/flow/scripts/install.py

```diff
@@ -325,23 +318,14 @@ def install(
         cli_symlink_dirs=cli_symdirs
     )
 
-    for entry_point in iter_entry_points(
-        'cylc.post_install'
-    ):
-        try:
-            entry_point.load()(
-                srcdir=source_dir,
-                opts=opts,
-                rundir=str(rundir)
-            )
-        except Exception as exc:
-            # NOTE: except Exception (purposefully vague)
-            # this is to separate plugin from core Cylc errors
-            raise PluginError(
-                'cylc.post_install',
-                entry_point.name,
-                exc
-            ) from None
+    async with async_block():
```

Comment:
This ensures that any async tasks started by the plugins are awaited before we move on.

Rose fileinstallation will create async tasks but cannot await them (because it's called synchronously) so we have to do it here, otherwise `cylc vip` would start running the workflow before local file installation has finished.